src/openzaak/conf/local_example.py

```diff
@@ -1,25 +1,13 @@
-import os
+import sys
 
-#
-# Any machine specific settings when using development settings.
-#
+from .dev import DATABASES, INSTALLED_APPS
```

Comment:
```suggestion
from .envs.dev import DATABASES, INSTALLED_APPS
```

Reply:
Should be fixed by changing.envs into .includes and swapping them